indico/modules/events/papers/util.py

```diff
@@ -85,5 +85,9 @@ def has_contributions_with_user_paper_submission_rights(event, user):
     return _query_contributions_with_user_paper_submission_rights(event, user).has_rows()
 
 
-def get_contributions_with_user_paper_submission_rights(event, user):
-    return _query_contributions_with_user_paper_submission_rights(event, user).all()
+def get_contributions_with_submission_rights_and_proper_state(event, user):
```

Comment:
Can't we find a nicer name for this? `get_user_submittable_contributions` isn't super nice either but doesn't sound as bad as both the previous name or the updated one.